react-components/src/metadata/admin/EditMetadataTemplate.js

```diff
@@ -105,9 +83,9 @@ class EditMetadataTemplate extends Component {
                         <IconButton id={build(ids.METADATA_TEMPLATE_FORM, ids.BUTTONS.CLOSE)}
                                     aria-label={formatMessage(intl, "close")}
                                     onClick={() => (
-                                        pristine ?
-                                            this.props.presenter.closeMetadataTemplateDialog() :
-                                            this.setState({showConfirmationDialog: true})
+                                        dirty ?
```

Comment:
Should this be `!dirty ? ...`  Maybe I'm misunderstanding the change from `pristine` to `dirty` though.  Below you have a similar change from `pristine` to `!dirty` in a button

Reply:
Ah, no, because the `then/else` lines have also swapped here 😉